### ISSUE TITLE
viewpager inside drawer

### ISSUE THREAD

**Issue by soheilserver** [mikepenz/MaterialDrawer#1114]
how is it possible to  have a viewpager with tablayout inside material drawer so that when it gets open, the user can swipe between some pages. thanx so much

**Comment by mikepenz** [comment]
@soheilserver could you please be a bit more clear about your requriement?

If the drawer is open there is no chance that the user can swipe in the area beside the drawer. a open draewr will take the whole screen and the area right beside the drawer will just catch the touch event and close the drawer when clicked. 

when you mean that you just want to navigate to a tab if an drawerItem is clicked, just use the `onItemClick` listener and select the tab you need

**Comment by soheilserver** [comment]
actually what i need is to have 2 tabs in the drawer. the swipe gesture is not important for me but i need 2 tabs so that when the user opens the drawer, he can easily access to two different pages in one drawer. is it possible to achieve such view in material drawer?
thank u for yout awesome library

**Comment by mikepenz** [comment]
@soheilserver yes it is possible, but it will require a lot of custom handling. 

Basically you will need to define a custom headerView. This header view has two buttons (your tabs) and if you click you switch the drawer content. 

You can also use the same mechanism used by the `AccountHeader` (as long as you only have 2 tabs)
https://github.com/mikepenz/MaterialDrawer/blob/develop/library/src/main/java/com/mikepenz/materialdrawer/Drawer.java#L927
https://github.com/mikepenz/MaterialDrawer/blob/develop/library/src/main/java/com/mikepenz/materialdrawer/Drawer.java#L956

https://github.com/mikepenz/MaterialDrawer/blob/develop/library/src/main/java/com/mikepenz/materialdrawer/DrawerBuilder.java#L548

**Comment by BillBosiolis** [comment]
Hi @soheilserver,

I had a similar requirement. What I finally did was to use a custom view.

To be more specific, I have a custom View which contains a TabLayout and a ViewPager. This View is used for navigational purposes. Currently it is being used for tablet layouts (as the left part of the screen while the right one is showing content based on the selections on the left part).

For phone support, I needed to show this 'menu' view as a drawer. So, I am using the setCustomView() method. In essence, I am not using much of the MaterialDrawer lib functionality in this case as all its content is provided by this custom view but still it saves you a looot of time.

I hope that helps.
